source/geometries/Next1EL.cc

```diff
@@ -955,8 +963,11 @@ void Next1EL::BuildFieldCage()
     new G4UnionSolid("SUPPORT_BAR", bar_base, addon, 0,
 		     G4ThreeVector(bar_thickn_, 0., (bar_length - bar_addon_length_)/2.));
 
+  G4Material* peek_mat_ = materials::PEEK();
```

Comment:
In [nexus conventions](https://github.com/next-exp/nexus/wiki/Coding-conventions-and-style-guide), we use the trailing underscore for members of the classes. In this case, since you are defining a local variable, you should just use `peek_mat`. Please, review the rest of the code following this convention.

Reply:
I have now added a commit with the changes.